openfl-tutorials/interactive_api/Tensorflow_Word_Prediction/envoy_folder/vectors/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pandas==1.3.3
+pyarrow=5.0.0
```

Comment:
This requirement looks unused

Reply:
Pandas uses it to save vectors. Now this part of code is moved from repo as the vectors are pre-calculated.